packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -271,4 +278,103 @@ export class ModelScene extends Scene {
       this.shadow.setSoftness(softness);
     }
   }
+
+  /**
+   * This method returns the world position and normal of the point on the
+   * mesh corresponding to the input pixel coordinates given relative to the
+   * model-viewer element. The position and normal are returned as strings in
+   * the format suitable for putting in a hotspot's data-position and
+   * data-normal attributes. If the mesh is not hit, position returns the
+   * empty string.
+   */
+  positionAndNormalFromPoint(pixelX: number, pixelY: number):
```

Comment:
Use your judgement, but I would recommend adding tests for all of this new API

Reply:
This method isn't new, it just moved; it's still tested by the annotation mixin. The new functions are very simple, so it would be more valuable to test them end-to-end. Once I get WebXR up to snuff, I'll look at enabling an Android device and then remake the AR tests to actually run and test these things.